### PR TITLE
Soojin Constrained Writing Documented in Personal Blog

### PR DESCRIPTION
**Pull request by Soojin-Lee0819**

### CODE REVIEW

File: 01-intro/README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ _(Please note you are welcome to post under a pseudonym and/or password protect
 
 - Name -- [assignment title](assignment url)
 - David Leyva -- [No Paris???](https://wp.nyu.edu/davidaleyva/week-1-homework/)
-- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9)
+- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [](https://soojin-lee0819.github.io/ITPBlog/week1.html)
```
Comment (shiffman) [inline 01-intro/README.md:60]: Hi Soojin!

With markdown it's best if you can put a "title" for the link in between the square brackets: `[insert title here]`.

```suggestion
- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [Translation with Numbers blog post](https://soojin-lee0819.github.io/ITPBlog/week1.html)
```

**Review by shiffman — COMMENTED** [review]
Great work @Soojin-Lee0819! I made a small comment with a suggestion, let me know what you think and then I can merge!

**Comment by Soojin-Lee0819** [comment]
Dear Prof. Daniel Shiffman

I will make changes.

Thank you for your suggestion!

Best,
Soojin

On Sun, 19 Sep 2021 at 8:56 AM Daniel Shiffman ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> Great work @Soojin-Lee0819
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Soojin-2DLee0819&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=6XhT1jCJfjr84UblN71H2aP4O-0c3IGRIVaosv8Oodc&e=>!
> I made a small comment with a suggestion, let me know what you think and
> then I can merge!
> ------------------------------
>
> In 01-intro/README.md
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_shiffman_A2Z-2DF21_pull_19-23discussion-5Fr711740793&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=vJK5R6lyXKIeqzomaegklJo8gHFB1fgBhAsSbLokSrM&e=>
> :
>
> > @@ -57,7 +57,7 @@ _(Please note you are welcome to post under a pseudonym and/or password protect
>
>  - Name -- [assignment title](assignment url)
>  - David Leyva -- [No Paris???](https://wp.nyu.edu/davidaleyva/week-1-homework/)
> -- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9)
> +- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [](https://soojin-lee0819.github.io/ITPBlog/week1.html)
>
> Hi Soojin!
>
> With markdown it's best if you can put a "title" for the link in between
> the square brackets: [insert title here].
> ⬇️ Suggested change
>
> -- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [](https://soojin-lee0819.github.io/ITPBlog/week1.html)
> +- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [Translation with Numbers blog post](https://soojin-lee0819.github.io/ITPBlog/week1.html)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_shiffman_A2Z-2DF21_pull_19-23pullrequestreview-2D758146997&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=27T-owARzuVXie3F7NimXjVMc2TQizAqzrAwELyFsZY&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AQ565TVFVSEFRHXWZRFF6UDUCXMV3ANCNFSM5EID6ZPA&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=16H0BZt_YHGOiIn3FQDtI82-ogN221QmoYPsx3_vNzk&e=>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__apps.apple.com_app_apple-2Dstore_id1477376905-3Fct-3Dnotification-2Demail-26mt-3D8-26pt-3D524675&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=6eySyFTi8ce0IKv_oBUNP3WvJB0UT14zribkGDlcQ4I&e=>
> or Android
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__play.google.com_store_apps_details-3Fid-3Dcom.github.android-26referrer-3Dutm-5Fcampaign-253Dnotification-2Demail-2526utm-5Fmedium-253Demail-2526utm-5Fsource-253Dgithub&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=r1xCuASdCjxDSskSKP9HJ75-OJUXQEkjOZYBNWTdeuU&e=>.
>
>
-- 
NYU Abu Dhabi, B.A., Class of 2023
Double Major in Interactive Media & Film and New Media
UAE Mobile: +971 54 348 3938
NYU Abu Dhabi, Saadiyat Campus
PO Box 129188
Abu Dhabi, United Arab Emirates

**Comment by Soojin-Lee0819** [comment]
Dear Professor Daniel Shiffman,

I read your comment on Github.

Thank you for adding the title for me.

And yes, please merge the section.

Thank you!.

Best regards,
Soojin

On Sun, Sep 19, 2021 at 10:24 AM Soojin Lee ***@***.***> wrote:

> Dear Prof. Daniel Shiffman
>
> I will make changes.
>
> Thank you for your suggestion!
>
> Best,
> Soojin
>
> On Sun, 19 Sep 2021 at 8:56 AM Daniel Shiffman ***@***.***>
> wrote:
>
>> ***@***.**** commented on this pull request.
>>
>> Great work @Soojin-Lee0819
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Soojin-2DLee0819&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=6XhT1jCJfjr84UblN71H2aP4O-0c3IGRIVaosv8Oodc&e=>!
>> I made a small comment with a suggestion, let me know what you think and
>> then I can merge!
>> ------------------------------
>>
>> In 01-intro/README.md
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_shiffman_A2Z-2DF21_pull_19-23discussion-5Fr711740793&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=vJK5R6lyXKIeqzomaegklJo8gHFB1fgBhAsSbLokSrM&e=>
>> :
>>
>> > @@ -57,7 +57,7 @@ _(Please note you are welcome to post under a pseudonym and/or password protect
>>
>>  - Name -- [assignment title](assignment url)
>>  - David Leyva -- [No Paris???](https://wp.nyu.edu/davidaleyva/week-1-homework/)
>> -- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9)
>> +- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [](https://soojin-lee0819.github.io/ITPBlog/week1.html)
>>
>> Hi Soojin!
>>
>> With markdown it's best if you can put a "title" for the link in between
>> the square brackets: [insert title here].
>> ⬇️ Suggested change
>>
>> -- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [](https://soojin-lee0819.github.io/ITPBlog/week1.html)
>> +- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [Translation with Numbers blog post](https://soojin-lee0819.github.io/ITPBlog/week1.html)
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_shiffman_A2Z-2DF21_pull_19-23pullrequestreview-2D758146997&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=27T-owARzuVXie3F7NimXjVMc2TQizAqzrAwELyFsZY&e=>,
>> or unsubscribe
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AQ565TVFVSEFRHXWZRFF6UDUCXMV3ANCNFSM5EID6ZPA&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=16H0BZt_YHGOiIn3FQDtI82-ogN221QmoYPsx3_vNzk&e=>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__apps.apple.com_app_apple-2Dstore_id1477376905-3Fct-3Dnotification-2Demail-26mt-3D8-26pt-3D524675&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=6eySyFTi8ce0IKv_oBUNP3WvJB0UT14zribkGDlcQ4I&e=>
>> or Android
>> <https://urldefense.proofpoint.com/v2/url?u=https-3A__play.google.com_store_apps_details-3Fid-3Dcom.github.android-26referrer-3Dutm-5Fcampaign-253Dnotification-2Demail-2526utm-5Fmedium-253Demail-2526utm-5Fsource-253Dgithub&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=r-2THZFdcM6LM3WHjdPMqA&m=P4p1RdeYwGdRXAsSHkrSUuEiCsix4_ebIQvRxercwQ0&s=r1xCuASdCjxDSskSKP9HJ75-OJUXQEkjOZYBNWTdeuU&e=>.
>>
>>
> --
> NYU Abu Dhabi, B.A., Class of 2023
> Double Major in Interactive Media & Film and New Media
> UAE Mobile: +971 54 348 3938
> NYU Abu Dhabi, Saadiyat Campus
> PO Box 129188
> Abu Dhabi, United Arab Emirates
>
>
>
>
>
>
>

-- 
NYU Abu Dhabi, B.A., Class of 2023
Double Major in Interactive Media & Film and New Media
UAE Mobile: +971 54 348 3938
NYU Abu Dhabi, Saadiyat Campus
PO Box 129188
Abu Dhabi, United Arab Emirates

**Comment by shiffman** [comment]
All set!

**Pull request merged**